lighthouse-core/gather/driver.js

```diff
@@ -395,6 +395,28 @@ class Driver {
   }
 
   /**
+  * @param {string} objectId Object ID for the resolved DOM node
+  * @param {string} propName Name of the property
+  * @return {!Promise<String>} The property value
+  */
+  getObjectProperty(objectId, propName) {
+    return new Promise(resolve => {
+      this.sendCommand('Runtime.getProperties', {
+        objectId,
+        accessorPropertiesOnly: true,
+        generatePreview: false,
+        ownProperties: false,
+      })
+      .then(properties => {
+        const propertyForName = properties.result
+          .find(property => property.name === propName)
```

Comment:
think we need to be more careful here. `.find()` returns `undefined` if the property name can't be found.